data_validation/validation_builder.py

```diff
@@ -397,37 +380,21 @@ def get_target_query(self):
             "data_client": self.target_client,
             "schema_name": self.config_manager.target_schema,
             "table_name": self.config_manager.target_table,
+            "target_query": self.config_manager.target_query,
         }
+
         if self.validation_type == consts.CUSTOM_QUERY:
-            target_input_query = self.get_query_from_file(
-                self.config_manager.target_query_file[0]
+            table = clients.get_ibis_query(
+                self.target_client, self.config_manager.target_query
             )
-            if self.config_manager.custom_query_type == "row":
-                calculated_query = CustomQueryBuilder().compile_custom_query(
-                    target_input_query, target_config
-                )
-                query = self.target_client.sql(calculated_query)
-            elif self.config_manager.custom_query_type == "column":
-                target_aggregate_query = "SELECT "
-                for aggregate in self.config_manager.aggregates:
-                    target_aggregate_query += (
-                        CustomQueryBuilder().get_aggregation_query(
-                            aggregate.get("type"), aggregate.get("target_column")
-                        )
-                    )
-                target_aggregate_query = (
-                    CustomQueryBuilder().get_wrapper_aggregation_query(
-                        target_aggregate_query, target_input_query
-                    )
-                )
-                query = self.target_client.sql(target_aggregate_query)
-            else:
-                raise ValueError(
-                    "Expected custom query type to be column or row, got an unacceptable value. "
-                    f"Input custom query type: {self.config_manager.custom_query_type}"
-                )
         else:
-            query = self.target_builder.compile(self.validation_type, **target_config)
+            table = clients.get_ibis_table(
+                self.target_client,
+                self.config_manager.target_schema,
+                self.config_manager.target_table,
+            )
+
+        query = self.target_builder.compile(self.validation_type, table)
         if self.verbose:
             logging.info(target_config)
             logging.info("-- ** Target Query ** --")
```

Comment:
We should remove the 'get_query_from_file` method on line 414 since you moved it to config_manager

Reply:
this is fixed. 
Had to remove testing of get_query_from_file() from  test-case of validation_builder.py and add it to test-case of config_manager.py